src/views/templates/actions/components/EditBootSourceModal.tsx

```diff
@@ -62,29 +64,28 @@ const EditBootSourceModal: FC<EditBootSourceModalProps> = ({
         isOpen={isOpen}
         onClose={onClose}
       >
-        <Alert isInline className="margin-bottom-md" variant="warning" title={t('Warning')}>
-          <Trans ns="plugin__kubevirt-plugin">
+        <Trans ns="plugin__kubevirt-plugin">
+          <Alert isInline className="margin-bottom-md" variant="warning" title={t('Warning')}>
             Editing the DataSource will affect{' '}
-            <Popover
-              headerContent={t('Affected templates')}
-              bodyContent={(affectedTemplates || []).map((template) => (
-                <ResourceLink
-                  key={template.metadata.uid}
-                  groupVersionKind={modelToGroupVersionKind(TemplateModel)}
-                  name={template.metadata.name}
-                  namespace={template.metadata.namespace}
-                  linkTo={false}
-                />
-              ))}
-            >
-              <Button variant="link" isInline>
-                {' '}
-                all templates
-              </Button>
-            </Popover>{' '}
+            <Button variant="link" isInline ref={popoverRef}>
+              all templates
+            </Button>{' '}
             that are currently using this DataSource.
-          </Trans>
-        </Alert>
+          </Alert>
+        </Trans>
+        <Popover
```

Comment:
Is there a benefit to using a ref for the Popover instead of wrapping the button with it?

Reply:
Yes. Wrapping the button with it - as it was originally here, didn't work to display the popover correctly.

Reply:
@hstastna I think we can do it without ref. If we need a ref it feels like the usage of the popover is missing something.. please check

Reply:
Yes. `Popover` component seems to not working 100% correctly. This seems to be related to Patternfly and changes of their components we use. And any other component around could influence that the original solution stopped working, not only Popover.. 
IMO fixing this can take some time, so I'd suggest to better go for the actual proposed and working solution, until we find some better way. IMO using ref is nothing bad here, just a different way that could be used.